### PR TITLE
Demo doc fixes.

### PR DESCRIPTION
**Pull request by setrofim**
- Add missing step to generate cbor CoRIMs.
- Amend paths to reflect new executable locations.

### CODE REVIEW

File: demo/psa/prov-verif-e2e/README.md [hunk 1]
```diff
@@ -82,7 +82,19 @@ Move to the docs/demo/psa/prov-verif-e2e directory:
 cd ${TOPDIR}/docs/demo/psa/prov-verif-e2e
 ```
 
-Ship the single CORIM that contains PSA reference values and trust anchor from data/cbor folder using following command
+Create a single CORIM that contains PSA reference values
```
[the remark this answers is not in the document]
Comment (yogeshbdeshpande) [reply demo/psa/prov-verif-e2e/README.md:85]: Please correct your folder locations in COCLI_README.md

[the remark this answers is not in the document]
Comment (setrofim) [reply demo/psa/prov-verif-e2e/README.md:85]: > These instructions are not needed as we expect the user to go via Route at line 18

Both Sabreen and I missed that. It's really not clear that  line 18 is a prerequisite for the demo. Bear in mind that the person following the steps may not be familiar with the documentation layout or terminology.  And besides, there something to be said for **having** a clear, linear sequence of steps, with shell snippets that the person following them can copy-paste from, without having that person jump all over the documentation. 

I've reworded to state that these instantiations are a duplication of those linked in section on line 18.

[the remark this answers is not in the document]
Comment (setrofim) [reply demo/psa/prov-verif-e2e/README.md:85]: > Please correct your folder locations in COCLI_README.md

Erm, please could you clarify what you mean? I don't see any services paths in COCLI_README.md that need to be fixed?

[the remark this answers is not in the document]
Comment (yogeshbdeshpande) [reply demo/psa/prov-verif-e2e/README.md:85]: Ok, I think, they refer to corim folders, so the comment is not really relevant:

**Review by yogeshbdeshpande — APPROVED** [review]

**Pull request merged**